model-api/bpmn-model/src/test/java/org/camunda/bpm/model/bpmn/builder/ProcessBuilderTest.java

```diff
@@ -103,9 +103,22 @@ public void testCreateEmptyProcess() {
       .hasSize(1);
 
     Process process = (Process) processes.iterator().next();
+
     assertThat(process.getId()).isNotNull();
   }
 
+  @Test
+  public void emptyProcessShouldHaveDefaultHTTL() {
```

Comment:
🔧 You could add another test for verifying that the default HTTL is not set when the `createProcess` is called with `false`.

Reply:
I've added 4 more tests for testing the `skipDefaultHistoryTimeToLive` with all values (`true`, `false`) against the methods `createProcess`, using a given `processId` or empty.